<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1183</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.ico</normaloff>icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="4" column="5">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>0 рублей 00 копеек</string>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>00:00:00</string>
        </property>
       </widget>
      </item>
      <item row="4" column="6">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Сумма за день:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Всего потрачено денег на проект:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="6">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Время за день:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="7">
       <widget class="QLabel" name="label_9">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>0 рублей 00 копеек</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4" colspan="4">
       <widget class="QPlainTextEdit" name="plainTextEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="7">
       <widget class="QLabel" name="label_8">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>00:00:00</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4" colspan="4">
       <widget class="QCalendarWidget" name="calendarWidget">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Всего потрачено времени на проект:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4" colspan="3">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Подробное описание задачи</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="3" colspan="4">
       <widget class="QTableWidget" name="tableWidget">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <column>
         <property name="text">
          <string>Задача</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Потраченное время</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Потраченные деньги</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="3" column="0" rowspan="2">
       <widget class="QPushButton" name="pushButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Новая задача</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>сompile/png_ico/new_task.png</normaloff>сompile/png_ico/new_task.png</iconset>
        </property>
       </widget>
      </item>
      <item row="3" column="1" rowspan="2">
       <widget class="QPushButton" name="pushButton_2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Запустить</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>png_ico/play.png</normaloff>png_ico/play.png</iconset>
        </property>
       </widget>
      </item>
      <item row="3" column="2" rowspan="2">
       <widget class="QPushButton" name="pushButton_3">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Остановить</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>png_ico/stop.png</normaloff>png_ico/stop.png</iconset>
        </property>
       </widget>
      </item>
      <item row="3" column="3" rowspan="2">
       <widget class="QPushButton" name="pushButton_4">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Удалить</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>png_ico/delete.png</normaloff>png_ico/delete.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <widget class="QCheckBox" name="checkBoxDone">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Выполнено</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Выберите папку проекта двойным щелчком ...</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1183</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Параметры проекта</string>
    </property>
    <addaction name="projectSettings"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Звонки по проекту</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Поиск</string>
    </property>
    <addaction name="action_3"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Отчёты</string>
    </property>
    <addaction name="action_5"/>
    <addaction name="action_6"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="projectSettings">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Управление проектом</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Создать звонок</string>
   </property>
  </action>
  <action name="action_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>История звонков</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Общий поиск</string>
   </property>
  </action>
  <action name="actionExcell">
   <property name="text">
    <string>Экспорт в Excell</string>
   </property>
  </action>
  <action name="actionCalc">
   <property name="text">
    <string>Экспорт в Calc</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Экспорт в текстовой файл</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>По времени</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>По выполненным задачам</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
